spec/models/rdv/send_webhook_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe Rdv, type: :model do
+  include ActiveJob::TestHelper
+
+  before(:each) do
```

Comment:
you can actually replace these three lines with 
`let!(:webhook_endpoint) { create(:webhook_endpoint) }`